ufw/config/applications.sls

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+# vim: ft=sls
+
+{#- Get the `tplroot` from `tpldir` #}
+{%- set tplroot = tpldir.split('/')[0] %}
+{%- set sls_package_install = tplroot ~ '.package.install' %}
+{%- set sls_enable_service  = tplroot ~ '.service.enable' %}
+{%- set sls_reload_service  = tplroot ~ '.service.reload' %}
+{%- from tplroot ~ "/map.jinja" import ufw with context %}
+
+include:
+  - {{ sls_package_install }}
+  - {{ sls_enable_service }}
+  - {{ sls_reload_service }}
+
+# Applications
+{%- for app_name, app_details in ufw.get('applications', {}).items() %}
+
+  {%- set from_addr_raw = app_details.get('from_addr', [None]) -%}
+  {%- set from_addrs = [from_addr_raw] if from_addr_raw is string else from_addr_raw -%}
```

Comment:
Need to be cautious with Jinja whitespace control `{%- ... -%}`: https://github.com/saltstack-formulas/collectd-formula/pull/141#issuecomment-485043263.  Both lines above.